lib/datadog/appsec/contrib/rails/framework.rb

```diff
@@ -17,7 +17,7 @@ def self.config_with_defaults(datadog_config)
 
           # Apply relevant configuration from Sinatra to Rack
           def self.activate_rack!(datadog_config, sinatra_config)
-            datadog_config.tracing.instrument(
+            datadog_config.instrument(
```

Comment:
My bad, this should have probably be named `appsec_config` to remove the ambiguity.